src/components/composer/composer/composer.ts

```diff
@@ -63,16 +66,23 @@ css/* scss */ `
 
         span {
           white-space: pre-wrap;
+          /* On some browsers (chromium ?) it is somehow possible to hover two of the composer's spans at the same time, leading to
+           * flickering with a succession of onmouseenter/onmouseleave events. A small invisible padding seems to prevent the issue.*/
+          padding-left: 0.01px;
 
           &.${selectionIndicatorClass}:after {
             content: "${selectionIndicator}";
             color: ${selectionIndicatorColor};
           }
 
-          &.${backgroundClass} {
+          &.${highlightParenthesisClass}:not(.${highlightClass}) {
             border-radius: 5px;
             background-color: lightgray;
-            padding: 0px 1.5px 1.5px 1.5px;
+            padding: 1.5px 0px 1.5px 0px;
+          }
+
+          &.${highlightClass} {
+            background-color: ${BUTTON_ACTIVE_BG};
```

Comment:
now that we can see how it looks, the background is not visible enough with this color IMO